docker/images/pinot-presto/README.md

```diff
@@ -34,9 +34,10 @@ This docker build project is specialized for Pinot.
 You can also build directly with `docker build` command by setting arguments:
 ```bash
 docker build \
-	--build-arg PRESTO_BRANCH=master \
+	--build-arg PRESTO_GIT_URL=$PRESTO_GIT_URL \
+	--build-arg PRESTO_BRANCH=$PRESTO_BRANCH \
 	--tag apachepinot/pinot-presto:latest \
-	--target build .
+	.
```

Comment:
Should we remove this dot?

Reply:
[New readme](https://github.com/yeongjukang/pinot/blob/patch-1/docker/images/pinot-presto/README.md) has proper dot. I think diff has problem on L39.